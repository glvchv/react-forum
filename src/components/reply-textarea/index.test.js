import React from 'react';
import renderer from 'react-test-renderer';
import ReplyTextArea from '.';

jest.mock('../error-message', () => 'ErrorMessage');

describe('Reply-textarea component', () => {
    it('should render component', () => {
        const tree = renderer.create(<ReplyTextArea handleReplyClick={() => {}}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});