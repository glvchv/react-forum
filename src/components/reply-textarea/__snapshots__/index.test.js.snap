// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reply-textarea component should render component 1`] = `
<div
  className="wrapper"
>
  <div
    className="form"
  >
    <h1
      className="title"
    >
      Reply
    </h1>
    <form
      className="form"
      onSubmit={[Function]}
    >
      <textarea
        className="text"
        id="reply-text-area"
        onChange={[Function]}
      />
      <ErrorMessage
        error=""
      />
      <button
        disabled={true}
        type="submit"
      >
        Submit
      </button>
    </form>
  </div>
</div>
`;
